net/net-snmp/files/snmpd.init

```diff
@@ -319,9 +440,11 @@ start_service() {
 	append_authtrapenable authtrapenable enable authtrapenable
 	append_parm v1trapaddress host v1trapaddress
 	append_parm trapsess trapsess trapsess
+	config_foreach snmpd_snmpv3_add v3 general
 
-	procd_set_param command $PROG -Lf /dev/null -f -r
-	procd_set_param file $CONFIGFILE
+	procd_set_param command $PROG -f -r -p "$pid_file"
+	procd_append_param command -C -c "$CONFIGFILE"
+	config_foreach snmpd_configure_logging log
```

Comment:
Why are we doing config file parsing so late?  Can we move this up above all of the `procd_*` stuff?

Reply:
This is about logging. If nothing is set in the uci config, then the old behaviour should be used again for the snmpd command call. Have a look into the new function snmpd_configure_logging https://github.com/openwrt/packages/pull/26918/files#diff-6382aa232061964385aa417cd8d13a4d3618d63a11768be4e831dc52527ed03aR402